package-lock.json

```diff
@@ -1,8 +1,7527 @@
 {
   "name": "npm-login-with-param",
   "version": "1.2.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
+      "version": "1.2.1",
+      "license": "ISC",
+      "dependencies": {
+        "commander": "^2.20.0"
+      },
+      "bin": {
+        "npm-login-with-param": "index.js"
+      },
+      "devDependencies": {
+        "eslint": "^5.16.0",
+        "eslint-config-airbnb-base": "^13.2.0",
+        "eslint-config-prettier": "^4.3.0",
+        "eslint-plugin-import": "^2.18.2",
+        "eslint-plugin-prettier": "^3.1.0",
+        "husky": "^2.7.0",
+        "jest-cli": "^24.9.0",
+        "prettier": "^1.18.2"
+      }
+    },
+    "node_modules/@babel/code-frame": {
+      "version": "7.0.0",
+      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.0.0.tgz",
+      "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
+      "dev": true,
+      "dependencies": {
+        "@babel/highlight": "^7.0.0"
+      }
+    },
+    "node_modules/@babel/core": {
+      "version": "7.6.0",
+      "resolved": "https://bluescape.jfrog.io/artifactory/api/npm/bluescape-repo/@babel/core/-/core-7.6.0.tgz",
```

Comment:
Whoops. This won't be resolvable by others outside of Bluescape

Reply:
The original copy I forked was already using Bluescape's internal (old) repo.
I just updated to our new repo.

Reply:
But I can make it use the public ones if that was the goal.

Reply:
I see. It's my mistake

Reply:
Done.